scripts/build/Jenkins/Jenkinsfile

```diff
@@ -534,6 +535,55 @@ def HandleDriveSnapshots(String repositoryName, String projectName, String pipel
     }
 }
 
+def ReplaceContent() {
+    // Get absolute directory for job metadata. Needs permissions from Jenkins for each upstream classes
+    def base_path = Jenkins.instance.getItemByFullName("${JOB_NAME}").rootDir
+    def log_file = "${base_path}" + "/builds/${env.BUILD_NUMBER}/log"
+    contentReplace(
+        configs: [
+            // Replace .cpp references to github links
+            fileContentReplaceConfig(
+            configs: [
+                fileContentReplaceItemConfig(
+                search: '( .*o3de)(.*\\.cpp)\\((\\d+)\\)',
+                replace: ' ' + GITHUB_URL + '$2#L$3',
+                matchCount: 0,
+                verbose: false,
+                )
+            ],
```

Comment:
This can be simplified by using a for loop. You can define a list of file extensions to replace and define another configs list and add each config in the for loop.